.github/workflows/ci.yaml

```diff
@@ -282,9 +273,11 @@ jobs:
           path: CHANGES.md
           if-no-files-found: error
 
+
   Release-GitHub:
     name: Release on GitHub
-    permissions: write-all
+    permissions:
+      contents: write
```

Comment:
According to https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#permission-on-contents this should be enough to write to the release endpoint.